recipes/watermark/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "watermark" %}
+{% set version = "1.3.1" %}
+{% set md5 = "cf25757d942e3a2ae5077b2ffff172dd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win]
```

Comment:
I just installed `watermark` via pip into a conda env on windows and it works, so I wouldn't skip windows in the build unless there is some other compelling reason.